test/parallel/test-global.js

```diff
@@ -40,7 +41,7 @@ assert.strictEqual(baseBar, // eslint-disable-line no-undef
                    'bar',
                    'global.x -> x in base level not working');
 
-const mod = require(path.join(common.fixturesDir, 'global', 'plain'));
+const mod = require(path.join(fixtures.fixturesDir, 'global', 'plain'));
```

Comment:
This could be using method `fixtures.path` instead of `path.join`.

Reply:
@pawelgolda is correct, the correct change here would be:

```js
const mod = require(fixtures.path('global', 'plain'));
```

Reply:
I am going to fix this while landing.